content/HTTPS is not enough - Pablo Blanco.md

```diff
@@ -0,0 +1,218 @@
+# HTTPS is not enough
+
+_[Pablo Blanco](https://www.rootstrap.com/blog/author/pabloblancorootstrapcom/)
+is Full-Stack developer on the Rootstrap team and writes about his experience
+with ethical hacking and information security. You can also follow Pablo on
+[LinkedIn](https://www.linkedin.com/in/pablo-blanco-a6b5a371/)._
+
+![White or black hat?](images/https-not-enough.jpg)
+
+Nowadays it is common to implement HTTPS for the sites and APIs we develop, well
+that’s fine but it is not enough to protect users from being hacked because
+there are chances of identity getting stolen, and an extra security layer should
+be applied. In this post, we are going to explain why only HTTPS is not enough
+to secure our sites and how can we improve user’s protection. But first, let’s
+recap with a few definitions.
+
+**What is TLS/SSL?**
+
+TLS (Transport Layer Security), formerly known as Secure Sockets Layer (SSL) is
+a standard security protocol for establishing an encrypted link between a server
+and a client. This is typically used to secure communications between web
+servers and a browser, or a mail server and a mail client.
+
+**What is HTTPS?**
+
+HTTPS (Hypertext Transfer Protocol Secure) is a secure version of the HTTP
+protocol that uses the SSL/TLS protocol for encryption and authentication.
+
+## Why only HTTPS is not enough?
+
+Encrypted communications are an essential requirement for health, banks,
+financial and other applications where information security is crucial.
+
+To improve accessibility, most secure websites run an HTTP(301) service to
+redirect users from HTTP to HTTPS endpoints. That redirection **does not** fully
+protect to the user since it can be intercepted between when the visitor
```

Comment:
```suggestion
redirect users from HTTP to HTTPS endpoints. That redirection **does not** fully
protect the user since it can be intercepted between when the visitor
```